src/store/reducers/settings.js

```diff
@@ -23,15 +25,17 @@ if (!preferences) {
 const settingsReducer = (state = initialState, { type, payload }) => {
   switch (type) {
     case SETTINGS_SET_AUTO_PLAY:
-      setPreferences({ ...state, autoPlay: payload });
+      setPreferences({ ...state.toJS(), autoPlay: payload });
```

Comment:
ugh... this `.toJS()` doesn't seem right.

Reply:
Yeah, but I was having problems serializing/deserializing from map into localstorage, I’ll have a second look later to see what can be done

Reply:
We could also update the localStorage `window.beforeunload`.